libs/dsb-address-book-nats-js/src/nats-jetstream-address-book.ts

```diff
@@ -78,6 +81,8 @@ export class NatsJetstreamAddressBook implements IAddressBook {
         });
 
         await this.transport.publish(this.addressBookChannel, 'default', claim);
+
+        this.cache.set(channel.fqcn, channel);
```

Comment:
i think this can be removed because in it will be pushed in sub callback

Reply:
Yes, but this also won't break, but can improve on cases where channel read is just after channel creation.